Gems/PhysX/Code/Include/PhysX/UserDataTypes.inl

```diff
@@ -9,9 +9,8 @@
 
 namespace PhysX
 {
-    // BaseActorData START ****************************************************
-    inline BaseActorData::BaseActorData(BaseActorType type, physx::PxActor* actor) :
-        m_sanity(s_sanityValue), m_actorType(type)
+    inline ActorData::ActorData(physx::PxActor* actor)
+        : m_sanity(s_sanityValue)
```

Comment:
minor: `m_sanity` assignment can be removed since it's done already in the declaration. Same in other constructors.